src/proper_parenthetics.py

```diff
@@ -0,0 +1,65 @@
+"""
+This module implements a doubly linked list and checks for values.
+
+We want to make sure every open parenthesis has a closing one.
+If there is an unmatched open paren, the function will return 1.
+If there is an unmatched closed paren, it will return -1.
+If all parens are matched in the doubly linked list, it returns 0.
+"""
+import sys
+
+class Node():
+    """Instantiate a node."""
+
+    def __init__(self, value=None, nxt=None, previous=None):
+        """."""
+        self.value = value
+        self.next = nxt
+        self.previous = previous
+
+
+class Queue():
+    """Instantiate a doubly linked list."""
+
+    def __init__(self, iter=None):
+        """."""
+        self.head = None
+        self.tail = None
+        self.length = 0
+        if sys.version_info[0] == 2:
+            if iter and isinstance(iter, str):
+                for item in list(iter):
+                    self.push(item)
+        elif sys.version_info[0] == 3:
+            if iter and hasattr(iter, "__iter__"):
+                for item in iter:
+                    self.push(item)
+        else:
+            raise TypeError
+
+    def push(self, value=None):
+        """Push value to the head of dll."""
+        new_node = Node(value, nxt=self.head)
+        if self.length < 1:
+            self.tail = new_node
+        else:
+            self.head.previous = new_node
+        self.head = new_node
+        self.length += 1
+
+    def is_balanced(self):
+        """Check for the number and place of parenthesis."""
+        counter = 0
+        curr = self.tail
+        while curr:
+            if curr.value == "(":
+                counter += 1
+            elif curr.value == ")":
+                counter -= 1
+            if counter < 0:
+                return -1
+            curr = curr.previous
+        if counter > 0:
+            return 1
+        if counter == 0:
```

Comment:
You don't need this last `if` block. If your method hasn't returned already, there's nothing else for it to do besides return `0`.